ff_derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fff_derive"
-version = "0.2.2"
+version = "0.2.1"
```

Comment:
please revert, this will be bumped when we do our release process

Reply:
Done as well.